pkg/cluster/spec/util.go

```diff
@@ -155,3 +155,9 @@ func MultiDirAbs(user, paths string) []string {
 	}
 	return dirs
 }
+
+// PackagePath return the tar bar path
+func PackagePath(comp string, version string, os string, arch string) string {
+	fileName := fmt.Sprintf("%s-%s-%s-%s.tar.gz", comp, version, os, arch)
+	return ProfilePath(TiOpsPackageCacheDir, fileName)
```

Comment:
Should we rename `TiOpsPackageCacheDir` to `TiUPPackageCacheDir`?

Reply:
good idea. TiOps is the predecessor of `tiup-cluster`. Since `tiup-dm` and `tiup-cluster` both use this variable, I think it should be `TiUPPackageCacheDir`